spring-boot-modules/pom.xml

```diff
@@ -63,10 +63,6 @@
         <module>spring-boot-nashorn</module>
         <module>spring-boot-parent</module>
         <module>spring-boot-performance</module>
```

Comment:
After merging conflicts, we may realize below removals aren't needed

Reply:
Yeah you are right, but that was the changes that I did 3 weeks ago, since then new changes was added.